src/GraphQL.Tests/GraphQL.Tests.csproj

```diff
@@ -21,6 +21,7 @@
 
   <ItemGroup>
     <ProjectReference Include="..\GraphQL.Dummy\GraphQL.Dummy.csproj" />
+    <ProjectReference Include="..\GraphQL.MicrosoftDI\GraphQL.MicrosoftDI.csproj" />
```

Comment:
What do you need from this package? ScopedAttribute?

Reply:
Yes